tests/base/utilities_pack_unpack_07.cc

```diff
@@ -61,6 +62,7 @@ check(const double (&array)[N])
     TimerOutput::Scope timer_section(computing_timer, "Pack uncompressed");
     Utilities::pack(array, array_uncompressed, false);
   }
+#endif
```

Comment:
I think the test for `Utilities::pack` in the uncompressed variant  should work even without `ZLIB` enabled.

Could you try out moving the `#endif` statement a few lines up so that it only embeds just the `default compressed` and `best compressed` variants?